src/tests/multihost/ad/test_adparameters.py

```diff
@@ -747,3 +748,40 @@ def test_0017_gssspnego_adjoin(self, multihost):
         remove_pcap = 'rm -f %s' % pcapfile
         multihost.client[0].run_command(remove_pcap)
         assert status == 'PASS'
+
+    @pytest.mark.tier1
+    def test_0018_bz1734040(self, multihost, adjoin):
+        """
+        @Title: ad_parameters: sssd crash in ad_get_account_domain_search
+        """
+        # Automation of bug
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1734040
+        adjoin(membersw='adcli')
+        client = sssdTools(multihost.client[0])
+        domain_name = client.get_domain_section_name()
+        cp = '/bin/cp -a /etc/sssd/sssd.conf /etc/sssd/sssd.conf.backup'
+        multihost.client[0].run_command(cp)
```

Comment:
Can we use client.backup_sssd_conf here and client.restore_sssd_conf on line 785,786?

Reply:
Updated the code with client.backup_sssd_conf and client.restore_sssd_conf.